iceoryx_utils/source/posix_wrapper/timer.cpp

```diff
@@ -76,52 +76,93 @@ void Timer::OsTimer::callbackHelper(sigval data)
         return;
     }
 
-    handle.m_timerInvocationCounter.fetch_add(1u, std::memory_order_relaxed);
```

Comment:
Logic change in this method together with start is supposed to address lost callbacks invocations.